static/js/publisher/release/components/releasesTable/cellViews.js

```diff
@@ -47,13 +47,66 @@ EmptyInfo.propTypes = {
   trackingChannel: PropTypes.string
 };
 
+const ProgressiveState = ({
```

Comment:
At first I thought it's rendering progress bar or something.
Name of the component doesn't suggest it's supposed to be used in tooltip - it would be good if it does. `ProgressiveTooltipMessage` maybe?